projects/js-packages/publicize-components/src/components/one-click-sharing-dropdown/index.js

```diff
@@ -0,0 +1,69 @@
+import { Text, ThemeProvider } from '@automattic/jetpack-components';
+import { Button, Dropdown } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/icons';
+import { useCallback } from 'react';
+import { ShareButtons } from '../share-buttons/share-buttons';
+import shareArrowIcon from './share-arrow-icon';
+import styles from './styles.module.scss';
+
+const renderToggle = ( { isOpen, onToggle } ) => {
+	return (
+		<Icon
+			className={ styles.icon }
+			size={ 18 }
+			icon={ shareArrowIcon }
+			onClick={ onToggle }
+			aria-expanded={ isOpen }
+		/>
+	);
+};
+
+const OneClickSharingDropdown = ( { className: containerClass, onClickLearnMore } ) => {
+	const renderContent = useCallback(
+		( { onToggle } ) => {
+			return (
+				<div className="one-click-sharing-dropdown">
+					<ThemeProvider>
```

Comment:
Is this required? Could we rely on the editor extension, or similar, being wrapped in this component?

Reply:
It is. We could move this into an outer component later, but without this the styles are not coming in

Reply:
Strange! I feel like we're using this in multiple places then and we'd be better off wrapping the whole sidebare in it. It must be used for the connection toggles for example. Something to look at.